queries/capnp/highlights.scm

```diff
@@ -68,8 +58,7 @@
 
 (enum_identifier) @type
 (extend_type) @type
-(field_type) @type
-(generic_identifier) @type
+; (generic_identifier) @type
```

Comment:
Why is this commented out?

Reply:
whoops, forgot to remove. generics *can* contain other types nested inside them, so it's redundant and would capture parentheses as `@type` sometimes.